### PR TITLE
Added Virtualization Setup for Fedora

### PR DESCRIPTION
**Pull request by Angxddeep**
# Pull Request

## Type of Change
- [x] New feature

## Testing
Works Completely

## Checklist
- [x] My code adheres to the coding and style guidelines of the project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generate no errors/warnings/merge conflicts.

### CODE REVIEW

File: tabs/system-setup/fedora/virtualization.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+            $ESCALATION_TOOL "$PACKAGER" install -y @virtualization 
+            echo "Installed virtualization tools..."
+            ;;
+        *)
+            echo "Unsupported distribution: $DTYPE"
+            ;;
+    esac
+}
+
+checkEnv
```
Comment (nnyyxxxx) [inline tabs/system-setup/fedora/virtualization.sh:19]: Call ``checkEscalationTool`` under ``checkEnv``

File: tabs/system-setup/fedora/virtualization.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#!/bin/sh -e
+
+. ../../common-script.sh
+
+# Install virtualization tools to enable virtual machines
+configureVirtualization() {
+    case $PACKAGER in
+        dnf)
+             echo "Installing virtualization tools..."
```
Comment (nnyyxxxx) [inline tabs/system-setup/fedora/virtualization.sh:9]:
```suggestion
            echo "Installing virtualization tools..."
```
There's an extra space here, everything needs to be formatted properly.

File: tabs/system-setup/fedora/virtualization.sh [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+            ;;
+        *)
+            echo "Unsupported distribution: $DTYPE"
+            ;;
+    esac
+}
+
+checkEnv () {
+    checkEscalationTool
+ }
```
Comment (nnyyxxxx) [inline tabs/system-setup/fedora/virtualization.sh:21]: Why is this here?

Comment (nnyyxxxx) [reply tabs/system-setup/fedora/virtualization.sh:21]:
```bash
checkEnv
checkEscalationTool
configureVirtualization
```

Comment (nnyyxxxx) [reply tabs/system-setup/fedora/virtualization.sh:21]: It should be formatted like this, do not make a separate function for checkEnv.

Comment (Angxddeep) [reply tabs/system-setup/fedora/virtualization.sh:21]: My bad

**Comment by nnyyxxxx** [comment]
@Angxddeep Mark all of these reviews as solved please.

**Review by nnyyxxxx — APPROVED** [review]
Looks good at the time of review.

**Comment by Angxddeep** [comment]
> Looks good at the time of review.

Yeah, we need to make it look good, although he's just gonna look at the code

File: tabs/system-setup/fedora/virtualization.sh [hunk 4]
```diff
@@ -0,0 +1,21 @@
+#!/bin/sh -e
+
+. ../../common-script.sh
+
+# Install virtualization tools to enable virtual machines
+configureVirtualization() {
+    case $PACKAGER in
+        dnf)
+            echo "Installing virtualization tools..."
```
Comment (nnyyxxxx) [inline tabs/system-setup/fedora/virtualization.sh:9]:
```suggestion
            printf "%b\n" ${YELLOW}"Installing virtualization tools...${RC}"
```

File: tabs/system-setup/fedora/virtualization.sh [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+            $ESCALATION_TOOL "$PACKAGER" install -y @virtualization 
+            echo "Installed virtualization tools..."
```
Comment (nnyyxxxx) [inline tabs/system-setup/fedora/virtualization.sh:11]:
```suggestion
            printf "%b\n" "${GREEN}Installed virtualization tools...${RC}"
```
